docker/Dockerfile

```diff
@@ -43,9 +43,7 @@ ENV UV_INDEX_STRATEGY="unsafe-best-match"
 # as it was causing spam when compiling the CUTLASS kernels
 RUN apt-get install -y gcc-10 g++-10
 RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 110 --slave /usr/bin/g++ g++ /usr/bin/g++-10
-RUN <<EOF
-gcc --version
-EOF
+RUN gcc --version
```

Comment:
fwiw heredocs is a supported by dockerfile frontend (which buildkit supports this by default)

but I also agree, for this no need to use heredoc